### PR TITLE
Send meta posts to slack

### PR DESCRIPTION
**Pull request by arfie**
This adds a slack message for meta posts, and slightly improves the formatting of the slack messages for unhandled replies.

### CODE REVIEW

File: tor/core/inbox.py [hunk 1]
```diff
@@ -22,12 +22,15 @@
...
 
 def forward_to_slack(item, config):
-    send_to_slack(
-        f'Unhandled inbox reply by *{item.author}* -- *{item.subject}*: '
-        f'{item.body}', config)
+    username = item.author.name
 
+    send_to_slack(
+        f'Unhandled message by '
+        f'<https://reddit.com/user/{username}|u/{username}> -- '
```
Comment (perryprog) [inline tor/core/inbox.py]: Why do you have the `{username}|u/{username}`? If it's for visual reasons, can you move the greater than sign to be after the first `{username}`?

File: tor/helpers/flair.py [hunk 2]
```diff
@@ -140,3 +140,6 @@ def set_meta_flair_on_other_posts(config):
                 f'Meta. '
             )
             flair_post(post, flair.meta)
+            send_to_slack(
+                f'New meta post: <{post.url}|{post.fullname}>'
```
Comment (perryprog) [inline tor/helpers/flair.py]: Same thing here. I'm going to assume I'm just missing something.

File: tor/core/inbox.py (hunk 1 above)
Comment (itsthejoker) [reply tor/core/inbox.py]: https://github.com/GrafeasGroup/tor/blob/master/tor/core/user_interaction.py#L53 This is how slack formats links. The first part is the actual link, the second part is how you want it to appear in slack. It's valid.

Comment (perryprog) [reply tor/core/inbox.py]: Figured

**Pull request merged**